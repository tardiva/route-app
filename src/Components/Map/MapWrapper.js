import makeAsyncScriptLoader from 'react-async-script';
import Map from './Map';

const key = 'AIzaSyCQfJgnHnO6LlY_DxE-VaYEB_qWV-W9O7c';
const url = `https://maps.googleapis.com/maps/api/js?key=${key}`;
const globalName = 'google';

export default makeAsyncScriptLoader(url, {
  globalName: globalName
})(Map);