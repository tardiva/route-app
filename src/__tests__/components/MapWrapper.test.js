import React from 'react';
import { shallow } from 'enzyme';
import MapWrapper from '../../Components/Map/MapWrapper';

it('should render MapWrapper correctly', () => {
  const wrapper = shallow(<MapWrapper />);
  expect(wrapper).toMatchSnapshot();
});