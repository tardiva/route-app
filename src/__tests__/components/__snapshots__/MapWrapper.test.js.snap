// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MapWrapper correctly 1`] = `
<AsyncScriptLoader
  forwardedRef={null}
/>
`;
